<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCustomer</class>
 <widget class="QDialog" name="AddCustomer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Додавання запису</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineSurname"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Прізвище:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Ім'я:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>По батькові:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineName">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Баланс:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="linePatronymic"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Місто:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Вулиця:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="cbMoney">
       <property name="text">
        <string>Додати гроші на рахунок</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spBalance">
       <property name="maximum">
        <number>1200</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Будинок:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="lineStreet"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Квартира:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QSpinBox" name="spApartment">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QSpinBox" name="spBuilding">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Тип:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QComboBox" name="cbConnectionTypes">
     <item>
      <property name="text">
       <string>Звита пара</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Оптоволокно</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QComboBox" name="cbTariffs">
     <item>
      <property name="text">
       <string>40 Мбіт/с</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>80 Мбіт/с</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>100 Мбіт/с</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Тариф:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbApply">
       <property name="text">
        <string>Додати</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>Відміна</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="cbCities">
     <item>
      <property name="text">
       <string>Маріуполь</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Покровськ</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Авдіївка</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineSurname</tabstop>
  <tabstop>lineName</tabstop>
  <tabstop>linePatronymic</tabstop>
  <tabstop>cbMoney</tabstop>
  <tabstop>spBalance</tabstop>
  <tabstop>cbCities</tabstop>
  <tabstop>lineStreet</tabstop>
  <tabstop>spBuilding</tabstop>
  <tabstop>spApartment</tabstop>
  <tabstop>cbConnectionTypes</tabstop>
  <tabstop>cbTariffs</tabstop>
  <tabstop>pbApply</tabstop>
  <tabstop>pbCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
